tools/ci_build/github/azure-pipelines/linux-cpu-aten-pipeline.yml

```diff
@@ -0,0 +1,67 @@
+jobs:
+- job: Linux_Build
+  timeoutInMinutes: 120
+  workspace:
+    clean: all
+  pool: Linux-CPU-2019
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '12.16.3'
+
+  - template: templates/get-docker-image-steps.yml
+    parameters:
+      Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_aten_cpu
+      Context: tools/ci_build/github/linux/docker
+      DockerBuildArgs: "--build-arg BUILD_UID=$( id -u )"
+      Repository: onnxruntimecpubuildaten
+
+  - task: CmdLine@2
+    displayName: 'build'
+    inputs:
+      script: |
+        mkdir -p $HOME/.onnx
+        docker run --rm \
+          --volume /data/onnx:/data/onnx:ro \
+          --volume $(Build.SourcesDirectory):/onnxruntime_src \
+          --volume $(Build.BinariesDirectory):/build \
+          --volume $HOME/.onnx:/home/onnxruntimedev/.onnx \
+          -e ALLOW_RELEASED_ONNX_OPSET_ONLY=0 \
+          -e NIGHTLY_BUILD \
+          -e BUILD_BUILDNUMBER \
+          onnxruntimecpubuildaten \
+            /opt/python/cp38-cp38/bin/python3 /onnxruntime_src/tools/ci_build/build.py \
+              --build_dir /build --cmake_generator Ninja \
+              --config Release \
+              --skip_submodule_sync \
+              --build_shared_lib \
+              --parallel \
+              --build_wheel \
+              --skip_test
+      workingDirectory: $(Build.SourcesDirectory)
+
+  - task: CmdLine@2
+    displayName: 'install ort_torch_ext and launch test'
+    inputs:
+      script: |
+        mkdir -p $HOME/.onnx
+        docker run --rm \
+          --volume /data/onnx:/data/onnx:ro \
+          --volume $(Build.SourcesDirectory):/onnxruntime_src \
+          --volume $(Build.BinariesDirectory):/build \
+          --volume $HOME/.onnx:/home/onnxruntimedev/.onnx \
+          -e ALLOW_RELEASED_ONNX_OPSET_ONLY=0 \
+          -e NIGHTLY_BUILD \
+          -e BUILD_BUILDNUMBER \
+          onnxruntimecpubuildaten \
```

Comment:
Don't run the test in manylinux docker image. Run it directly in the host OS: Ubuntu 20.04.  manylinux is specifically for constructing packaging pipelines, nobody uses it as a dev environment.  If the package you built from manylinux only works with manylinux docker image, then it has no real value to be published.

Reply:
I tried build the onnxuntime wheel in manylinux container, got the wheel from the container and installed it to my local Ubuntu machine, then installed the torch extension locally, it also works. I didn't see any ABI issue even the handle definition between onnxruntime and torch extension uses std::vector. To be safe, I modified the code not to use std::vector, but only use C objects. Regarding to the pipeline, is there any sample pipeline that run directly in host OS so I can leverage? I am not quite familiar with this pipeline setting, I added this ATen pipeline by copying most of the code/script from the eager mode pipeline and it also uses this manylinux image.